sycl/doc/EnvironmentVariables.md

```diff
@@ -25,6 +25,9 @@ compiler and runtime.
 | `SYCL_EAGER_INIT` | Integer | Enable by specifying non-zero value. Tells the SYCL runtime to do as much as possible initialization at objects construction as opposed to doing lazy initialization on the fly. This may mean doing some redundant work at warmup but ensures fastest possible execution on the following hot and reportable paths. It also instructs PI plugins to do the same. Default is "0".  |
 | `SYCL_REDUCTION_PREFERRED_WORKGROUP_SIZE` | See [below](#sycl_reduction_preferred_workgroup_size) | Controls the preferred work-group size of reductions. |
 | `SYCL_ENABLE_FUSION_CACHING` | '1' or '0' | Enable ('1') or disable ('0') caching of JIT compilations for kernel fusion. Caching avoids repeatedly running the JIT compilation pipeline if the same sequence of kernels is fused multiple times. Default value is '1'. |
+| `SYCL_JIT_KERNELS` | '1' or '0' | Enable ('1') or disable ('0') JIT compilation of kernels. Only supported for Nvidia and AMD backends. When JIT-ing is enabled SYCL runtime will try to cache and reuse JIT-compiled kernels, furthermore if a kernel uses specialization constants the compiler will attempt to materialize the values in place, turning them to de-facto compile time constants. Default is '0'. |
+| `SYCL_JIT_TARGET_CPU` | Any(\*) | Allows setting the target CPU to be used when JIT-ing kernels. Examples include setting SM version for Nvidia, or target architecture for AMD. |
+| `SYCL_JIT_TARGET_FEATURES` | Any(\*) | Allows setting desired target features to be used when JIT-ing kernels. Examples include setting PTX version for Nvidia. |
```

Comment:
I'm concerned that naming of those env variables does not reflect the scope where they can be applied and that may confuse someone.

Description for the last two doesn't even say that they are limited to certain backends, which could be perceived as it can changes something in JIT for Intel devices.

Reply:
Ended up spelling the arches, so we have now:

* `SYCL_JIT_AMDGCN_PTX_KERNELS`,
* `SYCL_JIT_AMDGCN_PTX_TARGET_CPU`,
* `SYCL_JIT_AMDGCN_PTX_TARGET_FEATURES`